file-formats/cdbo/cdbo/type/zif_aff_cdbo_v1.intf.abap

```diff
@@ -0,0 +1,155 @@
+INTERFACE zif_aff_cdbo_v1
+  PUBLIC.
+  "! <p class="shorttext">Object Type</p>
+  "! Object type
+  "! $values {@link zif_aff_cdbo_v1.data:co_object_type}
+  TYPES ty_object_type TYPE c LENGTH 2.
+
+  CONSTANTS:
+    "! <p class="shorttext">Object Type</p>
+    "! Object type
+    BEGIN OF co_object_type,
+      "! <p class="shorttext">Tables</p>
+      "! Tables
+      tables    TYPE ty_object_type VALUE '01',
+      "! <p class="shorttext">CDS Views</p>
+      "! CDS views
+      cds_views TYPE ty_object_type VALUE '02',
```

Comment:
This is coming from the Data Element Level

Reply:
> This is coming from the Data Element Level

What do you mean? How does the data element specifies values like "tables" or "cdsViews"?

I had following example in mind of object type `DMON`:
https://github.com/SAP/abap-file-formats/blob/883bfeb63046aca1654cf76bfc3c8606dfcf7cd1/file-formats/dmon/type/zif_aff_dmon_v1.intf.abap#L13-L26

However, you can still stick to your ABAP values `'01'` and `'02'`. 

Example (maybe, it must be `STOB` instead of `DDLS`?):

```suggestion
      "! <p class="shorttext">Tables</p>
      "! Tables
      "! $enumValue 'TABL'
      tabl    TYPE ty_object_type VALUE '01',
      "! <p class="shorttext">CDS Views</p>
      "! CDS views
      "! $enumValue 'DDLS'
      ddls TYPE ty_object_type VALUE '02',
```

Reply:
Actually, when we save the values in our DB then we need to Save it as 01 for table and 02 for STOB / DDLS that is the reason we have 01 and 02 as the values maintained.